### ISSUE TITLE
Add dynamic figures to the dashboard OG image

### ISSUE THREAD

**Issue by dadofsambonzuki** [teambtcmap/btcmap.org#144]

**Comment by stackingsaunter** [comment]
Do you have ideas for that? What are boundries. If you need some design let me know

**Comment by secondl1ght** [comment]
I dont think we need a design for this one since it will mostly just look like the actual dashboard on the site. But we could maybe use a design for the dynamic OG image for Merchant Pages. Unless we simply want to show a pin location on a map?

**Comment by dadofsambonzuki** [comment]
Agreed. The current design is fine, we just need the data to be dynamic.

#146 covers the dynamic merchant page OG feature. Comments to continue there.
